src/gui/hist_dialog.c

```diff
@@ -209,6 +214,12 @@ int dt_gui_hist_dialog_new(dt_gui_hist_dialog_t *d, int imgid, gboolean iscopy)
   gtk_tree_selection_set_mode(gtk_tree_view_get_selection(GTK_TREE_VIEW(d->items)), GTK_SELECTION_SINGLE);
   gtk_tree_view_set_model(GTK_TREE_VIEW(d->items), GTK_TREE_MODEL(liststore));
 
+  // we need access to the modules flags so ...
```

Comment:
I have a simpler solution where you could change the previous code with:

-      // This is sort of a hack, any better solution?
-      if (strcmp(item->op,"gamma") != 0)
+      if(!(get_module_flags(item->op) & IOP_FLAGS_HIDDEN))
       {

I'm about to push the commit to have the get_module_flags() routine.

Reply:
Thanks again.